### PR TITLE
Updates to chloroplast assembly tutorial

### PR DESCRIPTION
**Pull request by AnnaSyme**
Re-arranged content; added updated workflow; added updated JBrowse snippets

### CODE REVIEW

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md [hunk 1]
```diff
@@ -131,6 +132,8 @@ Your assembly graph may look like this:
 
 ![assembly graph](../../images/sweet-potato-assembly-graph.png "Assembly graph of the nanopore reads, for the sweet potato chloroplast genome. In your image, some of the labels may be truncated; this is a known bug under investigation.")
 
+*Note: a newer version of the Flye assembly tool now resolves this assembly into a single circle.*
```
Comment (hexylena) [inline topics/assembly/tutorials/chloroplast-assembly/tutorial.md:135]: Could you update the tool ID in the flye step to reflect this?

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md [hunk 2]
```diff
@@ -140,6 +143,7 @@ Your assembly graph may look like this:
 
 *Optional further steps:*
 * Repeat the Flye assembly with different parameters, and/or a filtered read set.
+* You can also try repeating the Flye assembly with an earlier version of the tool, to see the difference it makes. In the tool panel for Flye, click on the 'Versions' button at the top to change.
 * Try an alternative assembly tool, such as Canu or Unicycler.
```
Comment (hexylena) [inline topics/assembly/tutorials/chloroplast-assembly/tutorial.md:147]:
```suggestion
> ### {% icon comment %} Further Learning
> * Repeat the Flye assembly with different parameters, and/or a filtered read set.
> * You can also try repeating the Flye assembly with an earlier version of the tool, to see the difference it makes. In the tool panel for Flye, click on the 'Versions' button at the top to change.
> * Try an alternative assembly tool, such as Canu or Unicycler.
{: .comment} 
```

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md [hunk 3]
```diff
@@ -194,6 +197,62 @@ Next, we will compare the short reads to the assembly, and create a polished (co
...
+{: .hands_on}
+
+
+Alternatively, you might want to use a web-based tool designed for chloroplast genomes. 
+
+* One option is the GeSeq tool, described here. Skip this step if you have already used Prokka above. 
+
+> ### {% icon hands_on %} Hands-on: Annotate with GeSeq
+> * Download `polished-assembly.fasta` to your computer (click on the file in your history; then click on the disk icon).
+> * In a new broswer tab, go to [Chlorobox](https://chlorobox.mpimp-golm.mpg.de/geseq.html) where we will use the GeSeq tool {% cite Tillich_2017 %} to annotate our sequence.
```
Comment (hexylena) [inline topics/assembly/tutorials/chloroplast-assembly/tutorial.md:224]:
```suggestion
> * In a new broswer tab, go to [Chlorobox](https://chlorobox.mpimp-golm.mpg.de/geseq.html) where we will use the GeSeq tool ({% cite Tillich_2017 %}) to annotate our sequence.
```

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md [hunk 4]
```diff
@@ -194,6 +197,62 @@ Next, we will compare the short reads to the assembly, and create a polished (co
...
+> * Upload the `fasta` file there. Information about how to use the tool is available on the page.
+> * Once the annotation is completed, download the required files.
+> * In Galaxy, import the annotation `GFF3` file.
+{: .hands_on}
+
+Now make a JBrowse file to view the annotations (the GFF3 file - produced from either Prokka or GeSeq) under the assembly (the polished-assembly.fasta file).
+
+> ### {% icon hands_on %} Hands-on: View annotations
+> 1. {% tool [JBrowse genome browser](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.4+galaxy3) %}:
+>        - *"Reference genome to display"*: `Use a genome from history`
```
Comment (hexylena) [inline topics/assembly/tutorials/chloroplast-assembly/tutorial.md:234]: the indentation here looks a bit odd? Should this be with fewer spaces? And "select a reference" below indented below this element

**Review by hexylena — COMMENTED** [review]
This looks pretty good, just a couple of comments :)

**Comment by gallardoalba** [comment]
When running the website locally it shows this error: 
`'/training-material/topics/assembly/tutorials/chloroplast-assembly/data/names/c06/0.json' not found.`

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md (hunk 1 above)
Comment (AnnaSyme) [reply topics/assembly/tutorials/chloroplast-assembly/tutorial.md:135]: Yes - will change to (toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.6+galaxy0) - so version 2.6, and do you think that's correct with the extra "+galaxy0" part? That's what appears in the tool panel.

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md (hunk 4 above)
Comment (AnnaSyme) [reply topics/assembly/tutorials/chloroplast-assembly/tutorial.md:234]: Thanks for spotting this! will fix

**Comment by AnnaSyme** [comment]
> This looks pretty good, just a couple of comments :)

Thanks - have made some changes and accepted/committeed the suggestions above!

**Comment by AnnaSyme** [comment]
> When running the website locally it shows this error:
> `'/training-material/topics/assembly/tutorials/chloroplast-assembly/data/names/c06/0.json' not found.`

I also have a few json files with these errors but it doesn't seem to affect the local serve - do you notice any problems @hexylena ? If so I can investigate more ...

**Comment by hexylena** [comment]
@AnnaSyme do you need the feature search by name to work in the jbrowse? If not we can just remove the names directory and avoid this issue.

**Comment by AnnaSyme** [comment]
> @AnnaSyme do you need the feature search by name to work in the jbrowse? If not we can just remove the names directory and avoid this issue.

no need for feature search. thanks!

**Comment by hexylena** [comment]
Ok, I'll add on to this then!

**Review by hexylena — APPROVED** [review]

File: topics/assembly/tutorials/chloroplast-assembly/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/assembly/tutorials/chloroplast-assembly/tutorial.md:135]: Yep, that is right.

**Review by gallardoalba — APPROVED** [review]

**Comment by AnnaSyme** [comment]
Thanks for speedy review!

**Pull request merged**